FrontEnd/src/components/adminPage/table/ProfilesTable.jsx

```diff
@@ -0,0 +1,98 @@
+import { useState, useEffect } from 'react';
+import css from './ProfilesTable.module.css';
+import { useNavigate } from 'react-router-dom';
+import PaginationButtons from './PaginationButtons';
+import axios from 'axios';
+
+const COLUMN_NAMES = [
+    'ID', 'Person', 'Position', 'Company', 'Region', 'Phone', 'EDRPOU', 'Adress', 'IsDeleted', 'IsApproved'
+];
+
+function ProfilesTable() {
+    const [profiles, setProfiles] = useState([]);
+    const [loading, setLoading] = useState(true);
+    const [error, setError] = useState(null);
+    const [currentPage, setCurrentPage] = useState(1);
+    const [totalPages, setTotalPages] = useState(1);
+    const navigate = useNavigate();
+    const routeChange = (id) => {
+        let path = `../../customadmin/profile/${id}`;
+        navigate(path);
+    };
+    const [pageSize, setPageSize] = useState(10);
+    const handlePageSizeChange = (size) => {
+        setPageSize(size);
+        setCurrentPage(1);
+    };
+
+
+    useEffect(() => {
+        const fetchData = async () => {
+            try {
+                const response = await axios.get(
+                    `${process.env.REACT_APP_BASE_API_URL}/api/admin/profiles/?page=${currentPage}&page_size=${pageSize}`
+                );
+                if (response.status !== 200) {
+                    throw new Error(`HTTP error! Status: ${response.status}`);
+                }
+                setProfiles(response.data.results);
+                setTotalPages(response.data.total_pages);
+                setLoading(false);
+            } catch (error) {
+                setError(error.message);
+                setLoading(false);
+            }
+        };
```

Comment:
same we use an established pattern to fetch data using [swr](https://swr.vercel.app/)

Reply:
done